scripts/module/color.nut

```diff
@@ -6,12 +6,23 @@ function onPlayerCommand( player, cmd, text ) {
     local v = player.Vehicle;
     if ( !v ) {
       MessagePlayer( "[#ff0000]You're not in any vehicle." ,player );
-    } else if ( !text && !IsNum( text ) ) {
-      MessagePlayer( "[#ff0000]Usage: /color <Colour1ID>" ,player );
-    } else {
-      v.Colour1 = text.tointeger();
+      return 1;
     }
-    //FIXME: Support colour 2
+    if ( text ) {
+      local firstArg = GetTok(text, " ", 1);
+      if (IsNum(firstArg)) {
+        v.Colour1 = firstArg.tointeger();
+        if (NumTok(text, " ") >= 2) {
+          local secondArg = GetTok(text, " ", 2);
+          if (IsNum(secondArg)) {
+            v.Colour2 = secondArg.tointeger();
+          }
+        }
+        return 1;
+      }
+    }
+    MessagePlayer( "[#ff0000]Usage: /" + cmd + " <Colour1ID>, /" + cmd +
+                   " <Coulour1ID> <Colour2ID>", player );
```

Comment:
If only one colour is provided, it's probably a better idea to set both, color 1 and 2 to the same one.

It's what I most often want to do tbh.